myapp/spec/factories/articles.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :article do |f|
+    f.title "Testing"
```

Comment:
I recommend you use the https://github.com/stympy/faker gem to add random values